wcfsetup/install/files/lib/system/session/SessionHandler.class.php

```diff
@@ -201,38 +201,116 @@ public function setCookieSuffix() { }
 	public function setHasValidCookie($hasValidCookie) { }
 	
 	/**
-	 * Returns the session ID stored in the session cookie or `null`.
+	 * Parses the session cookie value, returning an array with the stored fields.
+	 * 
+	 * The return array is guaranteed to have a `sessionId` key.
+	 */
+	private function parseCookie(string $value): array {
+		$length = \mb_strlen($value, '8bit');
+		if ($length < 1) {
+			throw new \InvalidArgumentException(\sprintf(
+				'Expected at least 1 Byte, %d given.',
+				$length
+			));
```

Comment:
This could be simplified, `mb_strlen()` returns an integer and the `if` checks for "less than 1", therefore the only possible value is zero.

Reply:
I want to keep this for consistency with the version specific check in line 226.